test/e2e/dra/README.md

```diff
@@ -13,11 +13,55 @@ for the test driver and that the e2e test has full control over all gRPC calls,
 in case that it needs that for operations like error injection or checking
 calls.
 
-# Cluster setup
+## Cluster setup
 
 The container runtime must support CDI. The latest cri-o releases contain
 support, containerd 1.6.x does not. To bring up a kind cluster with containerd
 built from their main branch, use:
 
-    test/e2e/dra/kind-build-image.sh dra/node:latest && \
-    kind create cluster --config test/e2e/dra/kind.yaml --image dra/node:latest
+- Build node image
+
+```bash
+$ test/e2e/dra/kind-build-image.sh dra/node:latest
+```
+
+For macOS users, the step above can fail with the following error:
+
+```
+Starting to build Kubernetes
+ERROR: This script requires a minimum bash version of 4.2, but got version of 3.2
+On macOS with homebrew 'brew install bash' is sufficient.
+make: *** [quick-release-images] Error 1
+Failed to build Kubernetes: failed to build images: command "make quick-release-images 'KUBE_EXTRA_WHAT=cmd/kubeadm cmd/kubectl cmd/kubelet' KUBE_VERBOSE=0 KUBE_BUILD_HYPERKUBE=n KUBE_BUILD_CONFORMANCE=n KUBE_BUILD_PLATFORMS=linux/arm64" failed with error: exit status 2
+ERROR: error building node image: failed to build kubernetes: failed to build images: command "make quick-release-images 'KUBE_EXTRA_WHAT=cmd/kubeadm cmd/kubectl cmd/kubelet' KUBE_VERBOSE=0 KUBE_BUILD_HYPERKUBE=n KUBE_BUILD_CONFORMANCE=n KUBE_BUILD_PLATFORMS=linux/arm64" failed with error: exit status 2
+Command Output: ERROR: This script requires a minimum bash version of 4.2, but got version of 3.2
+```
```

Comment:
I don't think we need this. I'd suggest to expllain the requirement (bash version >= 4.2), how to check bash version and how to upgrade it.

Reply:
updated in 3b634de6ffc